x/crisis/keeper.go

```diff
@@ -49,4 +56,25 @@ func (k Keeper) Invariants() []sdk.Invariant {
 	return invars
 }
 
+// assert all invariants
+func (k Keeper) AssertInvariants(ctx sdk.Context, logger log.Logger) {
+
+	start := time.Now()
+	invarRoutes := k.Routes()
+	for _, ir := range invarRoutes {
+		if err := ir.Invar(ctx); err != nil {
+
+			// TODO make "gaiacli" app name a part of context to allow for this to be variable
```

Comment:
???

Reply:
so two lines later we can have 
`YOURAPPNAME tx crisis invariant-broken %v %v", err, ir.ModuleName, ir.Route))`
instead of `gaiacli`like it's hard set right now:
`gaiacli tx crisis invariant-broken %v %v", err, ir.ModuleName, ir.Route))`